### ISSUE TITLE
Dashboard: Clickable XCallback Links

### ISSUE THREAD

**Issue by dwertheimer** [NotePlan/plugins#468]
I have a bunch of clickable Xcallback links in my daily note, like:
```
* [Plan Tomorrow](noteplan://x-callback-url/runPlugin?pluginID=dwertheimer.TaskAutomations&command=Review%20Overdue%20Tasks%20as%20of%20%3CDate%3E&arg0=tomorrow) 
```
I would like to be able to click this link from the dashboard, but dashboard is removing the link. Is there a way we can keep it so that I can invoke commands from the dashboard which is telling me what i need to do?

**Comment by jgclark** [comment]
It's not deliberately removing them. I'm guessing it's an unwanted side-effect of the complex safety encode/decode needed over the interface. I'll have a look.

**Comment by jgclark** [comment]
Will be fixed in 0.6.0 from RC4
